app/layout/navbar.tsx

```diff
@@ -1,12 +1,9 @@
-import type { ReactNode } from "react"
-
 import { Navbar } from "~app/component/navbar"
 
-export function NavbarLayout(props: { children: ReactNode }) {
+export function NavbarLayout() {
```

Comment:
Do we need `NavbarLayout` or maybe we can just directly use `Navbar`? 
According to the design in figma, only info page includes this component.

Reply:
Another opinion is that layout without children is not a layout 😅 So instead of removing children, perhaps we could directly embed the `Navbar`